python/LoadExternalSTAC/LoadBiomassSTAC.ipynb

```diff
@@ -0,0 +1,250 @@
+      "text/plain": [
+       "<Figure size 600x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import rasterio\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "img = rasterio.open(\"biomass_elevation.tiff\").read()\n",
+    "biomass = img[0].flatten()\n",
+    "elevation = img[1].flatten()\n",
+    "\n",
+    "\n",
+    "f, ax = plt.subplots(figsize=(6, 6))\n",
+    "sns.scatterplot(x=elevation, y=biomass, s=5, color=\".15\")\n",
+    "sns.histplot(x=elevation, y=biomass, bins=50, pthresh=0.1, cmap=\"mako\")\n",
+    "sns.kdeplot(x=elevation, y=biomass, levels=5, color=\"w\", linewidths=1)\n",
+    "ax.set_xlabel(\"Elevation (m)\")\n",
+    "ax.set_ylabel(\"Biomass Carbon Density\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From this example plot we can see that in our study area the below-ground carbon biomass is mostly located at the elevation of 150-200 meters with value in the range of 200-400 kg/ha. Even though higher elevation also has biomass it is rarely below the range of 100 kg/ha. This shows an example of how we can combine different sources of data in OpenEO to perform a data assimilation and analysis.\n",
+    "\n",
+    "This example highlights the flexibility of openEO in accessing and combining external datasets via STAC. This was a simple example, but the same approach can be used for more complex analyses involving multiple datasets from different sources."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "openeo310",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.14"
+  }
+ },
+ "nbformat": 4,
```

Comment:
Very cool example! I really like it! perhaps only refer to the fed endpoint again